src/activeobject.h

```diff
@@ -78,20 +79,11 @@ enum ActiveObjectCommand {
 class ActiveObject
 {
 public:
-	ActiveObject(u16 id):
-		m_id(id)
-	{
-	}
+	ActiveObject(u16 id) : m_id(id) {}
 
-	u16 getId() const
-	{
-		return m_id;
-	}
+	u16 getId() const { return m_id; }
 
-	void setId(u16 id)
-	{
-		m_id = id;
-	}
+	void setId(u16 id) { m_id = id; }
```

Comment:
Changes like this are too opinionated.
What would be needed here is an option to allow both (as long as line lengths are respected).

Reply:
Agreed. While I personally prefer the one-liner, I wouldn't fuss too much about this.

Reply:
I don't see any way to let it be flexible in this

Reply:
This is fine.

Reply:
IMO the one-liner is much better and this is proper.